### DISCUSSION TITLE
t-range in visualization.plot_rotor_values

### DISCUSSION THREAD

**Discussion by zkdtxby** [NatLabRockies/floris#836] [Q&A] [answered]
How to provide a t-range parameter in visualization.plot_rotor_values, such as wind turbine numbers' F01 ',' F02 ',' F03 ',' F04 ',' F05 '.
![image](https://github.com/NREL/floris/assets/84053659/9998b233-9eb4-4c75-9a60-3653614dec82)

**Comment by misi9170** [comment, marked as answer]
Hi @zkdtxby , the error you are receiving is because you need to add a comma after the list, i.e., `t_range=[' ....'],` 
(at the moment, you don't have that comma, so python is not able to understand the difference between the `t_range` argument and the `return_fig_objects` argument).

However, the `wakeviz.plot_rotor_values` is not expecting a list of strings to be passed in. To achieve the behavior, you can specify `t_range` as a list of integer turbine indices, for example, `t_range = [0, 1, 2, 3, 4],`.

**Reply by zkdtxby** [reply]
Thank you very much, the issue has been resolved.
